src/style.css

```
@@ -1 +1,21 @@
-/* styles go here */
+.form-field {
+  display: block;
+  margin: 10px 0 0;
+}
+
+.fieldset {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
```

use only one-directional margins to avoid margin collapse
```suggestion
  margin-bottom: 20px;
}
```